etc/reset.py

```diff
@@ -19,6 +19,8 @@
     "secrets",
     "clusterrole",
     "clusterrolebinding",
+    "roles.rbac.authorization.k8s.io",
```

Comment:
I don't have context on why this change is necessary, but at first glance it seems fine.

Reply:
You can use `./etc/reset.py` to fully reset a development deployment of pachyderm. It looks like there were some recent changes on docker for mac that now make it necessary to delete roles and rolebindings, and it was blocking testing of these changes.